Lib/test/libregrtest/runtest.py

```diff
@@ -158,16 +167,24 @@ def findtestdir(path=None):
     return path or os.path.dirname(os.path.dirname(__file__)) or os.curdir
 
 
-def findtests(testdir=None, stdtests=STDTESTS, nottests=NOTTESTS):
+def findtests(testdir=None, stdtests=STDTESTS, nottests=NOTTESTS, splittestdirs=SPLITTESTDIRS, base_mod=""):
```

Comment:
spell that split_test_dirs= and lets make the new args keyword only.

Reply:
Done